src/index.html

```diff
@@ -9,8 +9,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+
+
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://www.google.com/"
+      method="post"
+      data-qa="big"
+      class="Big-input"
```

Comment:
```suggestion
      class="bigInput"
```
please use camelCase for your classes